docker-compose.dcproj

```diff
@@ -5,17 +5,13 @@
     <DockerLaunchBrowser>True</DockerLaunchBrowser>
```

Comment:
Is this file just for visual studio? Is there a way to run this from the command line tools?

Reply:
It's a visual studio project to run that enables you to run it in docker automatically.